docs/CLI.md

```diff
@@ -11,55 +11,8 @@ which is the place where the actual writing logic takes place.
 Installing
 ----------
 
-Head over to [etcher.io][etcher] and download the package that corresponds to
-your operating system.
-
-macOS and GNU/Linux
--------------------
-
-- Extract the `.tar.gz` package by running:
-
-```sh
-tar fvx path/to/cli.tar.gz
-```
-
-- Move the resulting directory to `/opt/etcher-cli`
-
-- Add `/opt/etcher-cli` to the `PATH`. For example, add the following to
-  `.bashrc` or `.zshrc`:
-
-```sh
-export PATH="$PATH:/opt/etcher-cli"
-```
-
-Windows
--------
-
-- Unzip the `.zip` package by right-clicking on it and selecting "Extract All"
-
-- Move the resulting directory to `C:\etcher-cli`
-
-- Add `C:\etcher-cli` to the `%PATH%`
-
-  - On Windows 10 and Windows 8
-    - Open *Control Panel*
-    - Open *System
-    - Click the *Advanced system settings* link
-    - Click *Environment Variables*
-    - Find the `PATH` environment variable, and click *Edit*
-    - Append `;C:\etcher-cli` to the environment variable value
-    - Click *OK*
-
-  - On Windows 7
-    - Right-click the *My Computer* icon
-    - Open the *Properties* menu
-    - Open the *Advanced* tab
-    - Click *Environment Variables*
-    - Find the `PATH` environment variable, and click *Edit*
-    - Append `;C:\etcher-cli` to the environment variable value
-    - Click *OK*
-
-  - Re-open `cmd.exe`, or PowerShell
+Head over to [etcher.io/cli][etcher-cli], download the package that corresponds to
```

Comment:
> Why is this pointing to etcher.io/cli

Because that's the URL that'll be hosting the download links ;-)

> Also is etcher.io/cli correct? There's nothing there (yet?)

See flowdock - @craig-mulligan is currently working on it...